reascripts/ReaSpeech/source/AlertPopup.lua

```diff
@@ -32,7 +32,12 @@ function AlertPopup:show(title, msg)
 end
 
 function AlertPopup:render_content()
-  ImGui.Text(ctx, self.msg)
+  if type(self.msg) == 'function' then
+    self.msg()
+    return
+  else
+    ImGui.Text(ctx, self.msg)
+  end
```

Comment:
This doesn't need to be in an else block

Reply:
🤦🏻‍♂️